src/citrine/informatics/predictors/graph_predictor.py

```diff
@@ -1,25 +1,35 @@
-from typing import List, Optional, Union
+from typing import List, Optional
 from uuid import UUID
 
+from citrine._rest.asynchronous_object import AsynchronousObject
 from citrine._rest.engine_resource import VersionedEngineResource
-from citrine._serialization import properties as _properties
+from citrine._serialization import properties as properties
+from citrine._session import Session
+from citrine._utils.functions import format_escaped_url
 from citrine.informatics.data_sources import DataSource
-from citrine.informatics.predictors import Predictor
+from citrine.informatics.predictors.single_predict_request import SinglePredictRequest
+from citrine.informatics.predictors.single_prediction import SinglePrediction
+from citrine.informatics.predictors import PredictorNode
 
 __all__ = ['GraphPredictor']
 
+from citrine.resources.report import ReportResource
 
-class GraphPredictor(VersionedEngineResource['GraphPredictor'], Predictor):
-    """A predictor interface that stitches other predictors together.
+
+class GraphPredictor(VersionedEngineResource['GraphPredictor'], AsynchronousObject):
+    """A predictor interface that stitches individual predictor nodes together.
+
+    The GraphPredictor is the only predictor that can be registered on the Citrine Platform
+    and carries along its meta-data regarding versioning, platform identifiers, and updates.
 
     Parameters
     ----------
     name: str
         name of the configuration
     description: str
         the description of the predictor
-    predictors: List[Union[UUID, Predictor]]
```

Comment:
Removing the `UUID` signature for graphs, which is technically a break.

Reply:
Where did we come down on the discussion of minor vs. major version bump?

Reply:
We didn't land on anything.

I suggested y'all have a look at this draft, which is the "ideal" state in terms of changes. And see if we can walk any of these changes back until we feel comfortable doing this under a minor instead of a major.

Reply:
I really doubt anyone is doing this, but we could keep this as `List[Union[UUID, Predictor]]`, and if there are any UUIDs filter them out and emit a deprecation warning

Reply:
That works. I can make that change.